app/api/crud.py

```diff
@@ -451,14 +451,22 @@ async def get_terms(
                 ),
                 None,
             )
-            term_label_dicts.append(
-                {
-                    "TermURL": util.replace_namespace_uri_with_prefix(
-                        term_url
-                    ),
-                    "Label": term_label,
-                }
+            matched_term = next(
+                (term for term in namespace_terms if term["id"] == term_id),
+                None,
             )
+            term_entry = {
+                "TermURL": util.replace_namespace_uri_with_prefix(term_url),
+                "Label": term_label,
+            }
+            if data_element_URI == "nb:Image":
+                term_entry["abbreviation"] = (
```

Comment:
I know it's not the same as what we have in the vocab files (which is not ideal), but let's keep the capitalization for the instance metadata consistent for now
```suggestion
                term_entry["Abbreviation"] = (
```
then: would you mind opening a separate API issue to consider standardizing the fields between the instances endpoints (/assessments, /diagnosis, etc.) and the vocabulary files in `communities`? My sense is we probably want to use the vocab file fields everywhere.